### PR TITLE
Fix some typos in English text

### PR DESCRIPTION
**Pull request by stweil**

### CODE REVIEW

File: site/en/_posts/2020-02-19-dhd.md [hunk 1]
```diff
@@ -8,4 +8,4 @@ author: Elisabeth Engl
...
 
-In V15, on March 5, the OCR-D coordination project will discuss the results and perspectives of the funding initiative on the basis of four theses on full text transformation. In the same section, the colleagues of the module project for the development of a model repository and automatic font recognition for OCR-D will report on their findings on the use of fracture in German-language books. For further information on the presentations see [the DHd programm](https://www.conftool.com/dhd2020/index.php?page=browseSessions&form_session=124&presentations=show).
+In V15, on March 5, the OCR-D coordination project will discuss the results and perspectives of the funding initiative on the basis of four theses on full text transformation. In the same section, the colleagues of the module project for the development of a model repository and automatic font recognition for OCR-D will report on their findings on the use of fracture in German-language books. For further information on the presentations see [the DHd program](https://www.conftool.com/dhd2020/index.php?page=browseSessions&form_session=124&presentations=show).
```
Comment (stweil) [inline site/en/_posts/2020-02-19-dhd.md:11]: programm -> program

File: site/en/faq.md [hunk 2]
```diff
@@ -91,7 +91,7 @@ tool offers simple ways to create new METS files or augment existing ones.
 OCR-D is Free Software, licensed under the terms of the Apache 2.0 license and
 will be free to use and adapt in perpetuity.
 
-### How are the full texts produced by OCR-D presented to the (library) user? Are they integrated into the library catalogue and can therefore be used for full text search in the catalogue?
+### How are the full texts produced by OCR-D presented to the (library) user? Are they integrated into the library catalog and can therefore be used for full text search in the catalog?
```
Comment (stweil) [inline site/en/faq.md:94]: The US form `catalog` is more common than the English `catalogue`. I kept the latter in names like the Union Catalogues of Books.

File: site/en/_posts/2020-02-19-dhd.md (hunk 1 above)
Comment (kba) [reply site/en/_posts/2020-02-19-dhd.md:11]: feel free to correct directly in the PR, thanks!

**Comment by stweil** [comment]
There are also English typos in old slides (site/slides) which could be fixed if required.

File: site/en/_posts/2020-02-19-dhd.md (hunk 1 above)
Comment (stweil) [reply site/en/_posts/2020-02-19-dhd.md:11]: My comment was just to indicate what was changed in that lengthy text.

Comment (kba) [reply site/en/_posts/2020-02-19-dhd.md:11]: Right, sry, I had only seen the e-mail.

**Pull request merged**